### PR TITLE
Data Release: 2020-04-04

### PR DESCRIPTION
**Pull request by shifucun**
Also updated proto to reflect stat var definition change

### CODE REVIEW

File: internal/server/statvar/golden/get_statvar_group_node/school.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "searchNames": [
         "Population Count of Person With Educational Attainment is 1st Grade"
       ],
-      "displayName": "1st Grade Population",
+      "displayName": "Population: 1st Grade",
+      "definition": "mp=count,pt=Person,educationalAttainment=1StGrade",
```
Comment (shifucun) [inline internal/server/statvar/golden/get_statvar_group_node/school.json:11]: @nopper This is the new field add for stat var definition

File: internal/server/node/golden/get_triples/stat_var.json [hunk 2]
```diff
@@ -96,7 +96,7 @@
     },
     {
       "subjectId": "Amount_EconomicActivity_ExpenditureActivity_HealthcareExpenditure_AsFractionOf_Count_Person",
-      "subjectName": "Amount of Expenditure Activity, Healthcare Expenditure Per Capita",
+      "subjectName": "Amount of Economic Activity: Expenditure Activity, Healthcare Expenditure (Per Capita)",
```
Comment (pradh) [inline internal/server/node/golden/get_triples/stat_var.json:99]: These name changes are expected

File: internal/server/stat/golden/get_stat_set_series/weather.json [hunk 3]
```diff
@@ -1143,684 +1143,1478 @@
...
+            "2021-06": 43.9,
+            "2021-07": 45.6,
+            "2021-08": 42.2,
+            "2021-09": 42.222,
+            "2021-10": 34.4,
+            "2021-11": 28.9,
+            "2021-12": 26.7,
+            "2022-01": 21.7,
+            "2022-02": 30,
+            "2022-03": 26.722
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_set_series/weather.json:2612]: We now have much longer observation monthly weather, as expected.

File: internal/server/stat/golden/get_stat_set_within_place/US_State.json [hunk 4]
```diff
@@ -526,263 +526,263 @@
     "Count_Person_Employed": {
       "stat": {
         "geoId/01": {
-          "date": "2021-12",
-          "value": 2190745,
+          "date": "2022-02",
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_set_within_place/US_State.json:529]: New BLS data refresh.

File: internal/server/v0/statvarsummary/golden/get_statvar_summary/covid.json [hunk 5]
```diff
@@ -17,7 +17,9 @@
               "dcid": "geoId/2937592",
               "name": "Joplin"
             }
-          ]
+          ],
+          "minValue": 1,
```
Comment (pradh) [inline internal/server/v0/statvarsummary/golden/get_statvar_summary/covid.json:21]: Expected

File: internal/server/v0/statpoint/golden/get_stat_value/umemployed.json [hunk 6]
```diff
@@ -1,3 +1,3 @@
 {
-  "value": 6270000
+  "value": 5952000
```
Comment (pradh) [inline internal/server/v0/statpoint/golden/get_stat_value/umemployed.json:2]: Likely from BLS refresh.  (Might be nice to tell from the path of the file, or some other way what the place is)

File: internal/server/v0/internalplace/golden/get_place_page_data/tha.sample.json [hunk 7]
```diff
@@ -9817,741 +9910,831 @@
         },
         "CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased": {
           "val": {
+            "2020-01-03": 0,
```
Comment (pradh) [inline internal/server/v0/internalplace/golden/get_place_page_data/tha.sample.json:9913]: Looks like WHO Covid dataset has added some older date obs

**Review by pradh — APPROVED** [review]
Generally looks good to me!

**Comment by shifucun** [comment]
Thanks for adding notes for the diff!

**Pull request merged**